File: src/main/java/io/horacerta/model/Pessoa.java
package io.horacerta.model;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.ForeignKey;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

/**
 * @author Pedro Ribeiro
 *
 */
@Entity
public class Pessoa {

   @Id
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   private Long id;
   
   private String nome;
   
   private String cargo;
   
   @Temporal(TemporalType.DATE)
   private Date dataRegistro = new Date();

   private String username;

   public Long getId() {
      return id;
   }

   public void setId(Long id) {
      this.id = id;
   }

   public String getNome() {
      return nome;
   }

   public void setNome(String nome) {
      this.nome = nome;
   }

   public String getCargo() {
      return cargo;
   }

   public void setCargo(String cargo) {
      this.cargo = cargo;
   }

public String getUsername() {
	return username;
}

public void setUsername(String username) {
	this.username = username;
}

}

File: frontend/horacerta/src/app/model/pessoa.ts
export class Pessoa {
    public id: Number;
    public nome: String;
    public cargo: String;
    public dataRegistro: Date;
    public username: String;
    public telefone: String;

    constructor(id: Number, nome: String, cargo: String, dataRegistro: Date, username: String, telefone: String) {
        this.id = id;
        this.nome = nome;
        this.cargo = cargo;
        this.dataRegistro = dataRegistro;
        this.username = username;
        this.telefone = telefone;
    }

}